{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/apps/core/models.py

```diff
@@ -10,9 +10,9 @@ class User(AbstractUser):
     """
     Custom user model for use with OpenID Connect.
 
-    .. pii:: Stores full name, username, and email address for a user.
-    .. pii_types:: name, other
-    .. pii_retirement:: local_api
+    .. pii: Stores full name, username, and email address for a user.
+    .. pii_types: name, other
```

Comment:
Nit: Care to right my wrong here? And change `pii_types` to `name, username, email_address`?

Reply:
@doctoryes all set